frontend/src/components/auth/TemporaryPasswordReroute.js

```diff
@@ -0,0 +1,141 @@
+import { Link } from 'react-router-dom'
+import { login, verify, getFPByEmail, getPMByEmail } from '../../utils/ApiWrapper'
+import { Form, Button, FormGroup, Input, Card, CardBody } from 'reactstrap'
+import { setCookie } from '../../utils/cookie'
+import React, { Component } from 'react'
+import BackgroundSlideshow from 'react-background-slideshow'
+import Navbar from '../NavBar'
+
+import '../../styles/index.css'
+import '../../styles/login.css'
+import '../../styles/navbar.css'
+
+import b1 from '../../media/b1-min.jpg'
+import b3 from '../../media/b3-min.jpg'
+import b4 from '../../media/b4-min.jpg'
+import b5 from '../../media/b5-min.jpg'
+import b6 from '../../media/b6-min.jpg'
+import kivaLogo from '../../media/kivaPlainLogo.png'
+
+const EMAIL_REGEX =
+  "([a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)@([a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+).([a-zA-Z]{2,3}).?([a-zA-Z]{0,3})"
+// const PASSWORD_REGEX = "^(((?=.*[a-z])(?=.*[A-Z]))|((?=.*[a-z])(?=.*[0-9]))|((?=.*[A-Z])(?=.*[0-9])))(?=.{6,})";
+
+/**
+ * This is the login page.
+ * It contains a form that takes in user email and password
+ * In case of forgotten password, it has a Forgot Password button that leads to a recovery page
+ */
+class LogIn extends Component {
```

Comment:
I would actually modify this class name as well as the comments since this is a copy paste right, and we don't want the temporary page to have the exact same name as the actual login page